.github/workflows/examples.yml

```diff
@@ -10,6 +10,12 @@ on:
     paths:
       - 'examples/**'
   workflow_dispatch:
+    inputs:
+      tag:
+        description: 'The tag to use for the OCI artifacts.'
+        required: false
```

Comment:
Nit: Should this be `required: true`? The default value means no one will need to fill something out, but the required means that the field won't be empty. I know `${{ steps.meta.outputs.release_tag || 'latest' }}` will catch this as well, which is why this is also a nit

Reply:
Good call, thanks! We really want a tag specified if workflow_dispatch is used, anyway. 99% of the time it will be "latest".